api/src/certification/complementary-certification/infrastructure/repositories/complementary-certification-for-admin-repository.js

```diff
@@ -0,0 +1,24 @@
+import { ComplementaryCertificationForAdmin } from '../../domain/models/ComplementaryCertificationForAdmin.js';
```

Comment:
Pareil, ici pas de raison de mettre "for-admin" dans le nom du fichier. On a pas de repo pour les complementary-certification dans ce sous-contexte.

Reply:
vu ensemble, effectivement tout à été déplacé par après donc ça n'a plus de sens de garder ce nom